source/go-perun/new_tutorial/client/handle.rst

```diff
@@ -0,0 +1,138 @@
+.. _client-handle:
+
+Handler
+-------
+As mentioned in the Client section, go-perun uses callbacks to handle incoming channel proposals and channel update requests.
+The callbacks are managed via the `handler`, that comes with the Perun Client.
+The following code is located in `client/handle.go`.
+
+Handling Channel Proposals
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+`HandleProposal()` is triggered on incoming Channel proposals.
+Before any Channel is accepted, it is essential to validate its "terms and conditions".
+
+In our case, the Client expects the following properties to be met:
+
+#. The proposed payment-channel must be a standard `LedgerChannel`, therefore the proposal is expected to be of type `LedgerChannelProposal`.
```

Comment:
```suggestion
#. The proposed payment-channel must be a standard `LedgerChannel`, therefore the proposal is expected to be of type `LedgerChannelProposalMsg`.
```

This has changed recently. Depends on the go-perun version on which is correct here. The upcoming release 0.9.0 planned for February will contain the change.